### DISCUSSION TITLE
Paperdoll depending on classes and gender

### DISCUSSION THREAD

**Discussion by Gibier1** [AscensionGameDev/Intersect-Engine#1787] [Ideas]
Did it would make more sense to base paperdoll on classes too? For many games each classes don't use the same sprite so you need to create different version of the same item.

**Comment by Gibier1** [comment]
Something like this would be amazing

![image](https://user-images.githubusercontent.com/98491438/232316392-7baffdb0-f438-4e11-b336-582b02bf9ae2.png)

**Comment by SilverShardOnline** [comment]
Looks like there are multiple threads asking for the same feature.
Would help if everyone voted on the same one to give it more attention.

I believe this is a very simple change that has a very large impact, and deserves more attention.
